release.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -xe
+rm -rf dist/
+mkdir dist
+
+mvn clean package -DskipTests
```

Comment:
Can we run the tests? I'm not a fan of building something without running the tests.